packages/core/core/src/Parcel.js

```diff
@@ -389,7 +389,7 @@ export default class Parcel {
     let resolvedOptions = nullthrows(this.#resolvedOptions);
     let opts = getWatcherOptions(resolvedOptions);
     let sub = await resolvedOptions.inputFS.watch(
-      resolvedOptions.projectRoot,
+      resolvedOptions.watchDir,
       (err, events) => {
         if (err) {
```

Comment:
we should probably also update this so it works for one-shot builds https://github.com/parcel-bundler/parcel/blob/e4f0a69ac764ef927749a1392e2834af8b0bef99/packages/core/core/src/RequestTracker.js#L1217

idk if watchRoot is the most obvious name in that case but not sure about another one. projectRoot is a different thing and affects more stuff so would need to be something different.

Reply:
Done! Yeah the name because less intuitive when used in this context :|